train_vnext/training.py

```diff
@@ -155,38 +156,36 @@ def process_post_training_csv(csv_path, training_id, classification_name_to_clas
 
 def save_training_session(config, model_location, perf_location, prediction_labels_location):
     # First, upload the model to blob storage
-    cur_date_time = "{date:%Y_%m_%d_%H_%M_%S}".format(date=datetime.datetime.now())
+    cur_date_time = "{date:%Y_%m_%d_%H_%M_%S}".format(date=datetime.datetime.utcnow())
     file_name = "frozen_inference_graph_" + cur_date_time + ".pb"
-    model_location = upload_model_to_blob_storage(config, model_location, file_name)
+    model_location = upload_model_to_blob_storage(config, model_location, file_name, config.get("tagging_user"))
 
     # Get the mapping for class name to class id
-    overall_average, classification_name_to_class_id = get_average_and_classification_map(perf_location, config.get("tagging_user"), config.get("url"))
+    overall_average, classification_name_to_class_id, avg_dictionary = process_classifications(perf_location, config.get("tagging_user"), config.get("url"))
 
     # Create a new training session in db and get its id
-    training_id = construct_new_training_session(perf_location, classification_name_to_class_id, overall_average, cur_date_time, model_location, config.get("tagging_user"), config.get("url"))
+    training_id = construct_new_training_session(perf_location, classification_name_to_class_id, overall_average, cur_date_time, model_location, avg_dictionary, config.get("tagging_user"), config.get("url"))
 
     # Upload prediction labels to the db
     upload_data_post_training(prediction_labels_location, classification_name_to_class_id, training_id, config.get("tagging_user"), config.get("url"))
 
-def upload_model_to_blob_storage(config, model_location, file_name):
+def upload_model_to_blob_storage(config, model_location, file_name, user_name):
     blob_storage = BlobStorage.get_azure_storage_client(config)
+    blob_metadata = {
+            "userFilePath": model_location,
```

Comment:
@samiyaakhtar I'm not sure how useful the userFilePath for the model on the training machine will be. I have been thinking we should save the config.ini (it will have that location as well) somewhere but that is out of scope. Storing the training id would be more useful right now

Reply:
@samiyaakhtar got it. My bad.